package-templates/react/package.json

```diff
@@ -22,12 +22,18 @@
     "reinstall": "rm -rf node_modules package-lock.json && npm install",
     "prepublishOnly": "npm run check && npm run build",
     "check": "npm run lint && npm run test",
-    "lint": "eslint -f codeframe \"./**/*.{js,jsx}\"",
+    "check:tofile": "npm run lint:tofile; npm run test:tofile",
+    "eslint": "eslint -f codeframe \"./**/*.{js,jsx}\"",
+    "eslint:tofile": "npm run eslint -- --format junit -o ../../reports/react-template-eslint.xml",
+    "stylelint": "stylelint \"./**/*.scss\" ",
+    "stylelint:tofile": "npm run stylelint -- --custom-formatter '../../node_modules/stylelint-junit-formatter' -o ../../reports/react-template-stylelint.xml",
```

Comment:
Will try to run all those check:tofile scripts directly in root package so that we don't have to copy&paste this mess into every subpackage

Reply:
Stylelint works in root directory but eslint doesn't. Throws some weird internal error.

Reply:
might be related to node_modules/eslintignore configuration

Reply:
I finally moved everything into the root package. Works great, way less noise in subpackages. 

(Take a look at .eslintrc)